### ISSUE TITLE
Ledger device: UNKNOWN_ERROR (0x6a82)

### ISSUE THREAD

**Issue by aphex3k** [LedgerHQ/ledger-live-desktop#4473]
#### Ledger Live Version and Operating System

- tested on Ledger Live 2.35.2
    - Ledger Nano S ₿itcoin app 2.0.0
- Platform and version: **macOS Monterey 12.0.1**

#### Expected behavior

I expect to connect to remote bitcoind full node to create lss.json for SatStack.

#### Actual behavior

`Ledger device: UNKNOWN_ERROR (0x6a82)`

#### Steps to reproduce the behavior

- I tried different USB/Thunderbolt adapters/cables
- Other functionality during regular Ledger Live use seems unaffected

<img width="595" alt="Screen Shot 2021-11-26 at 2 10 17 PM" src="https://user-images.githubusercontent.com/433270/143658363-2c8b4e36-e7ee-4a9e-8b2a-196756fd476c.png">

**Comment by Retropex** [comment]
Same problem for myself after several searches I cannot find the solution.

**Comment by premmen2** [comment]
Same for me on Windows 11 with the latest version of Ledger Live and the bitcoin app.

**Comment by Retropex** [comment]
it's good I have the solution just downgrade the version of ledger live for it to work, the latest version breaks ledger sat stack. use version 2.33.1

**Comment by aphex3k** [comment]
> it's good I have the solution just downgrade the version of ledger live for it to work, the latest version breaks ledger sat stack. use version 2.33.1

Where do I find an old version for Ledger Live in general and for macOS specifically?

**Comment by Retropex** [comment]
> > it's good I have the solution just downgrade the version of ledger live for it to work, the latest version breaks ledger sat stack. use version 2.33.1
> 
> Where do I find an old version for Ledger Live in general and for macOS specifically?

all versions for all platforms are available here : https://github.com/LedgerHQ/ledger-live-desktop

**Comment by d-rez** [comment]
Same issue here with a Nano X. Zzzz. I don't mind temporarily not using my node but can this be fixed, Ledger? Or at least acknowledged? I noticed something wasn't right when it stopped syncing data with the BTC network recently.

Ledger live v2.36.2
BTC App v2.0.1

**Comment by aphex3k** [comment]
It works if you reverts back to an older Ledger version, however, you lose Taproot support.

**Comment by plk** [comment]
Same issue here with OSX, 2.36.2. Would be very nice to have this fixed so we can have Taproot and full node support. Makes the documentation page on sovereignty and verification look a little embarrassing since this hasn't worked for a few versions now ...

**Comment by tyuxar** [comment]
I ran into the same bug on macOS 10.15.7 + Ledger Live 2.36.3.

I also found a "workaround": turn off the "Experimental JS implementations" in Settings, then it worked for me. Obviously this is only theoretical, there's not taproot support without it.

**Comment by d-rez** [comment]
> I also found a "workaround": turn off the "Experimental JS implementations" in Settings, then it worked for me. Obviously this is only theoretical, there's not taproot support without it.

That doesn't seem like a viable workaround given that I never had this option enabled in the first place :D

**Comment by tyuxar** [comment]
> > I also found a "workaround": turn off the "Experimental JS implementations" in Settings, then it worked for me. Obviously this is only theoretical, there's not taproot support without it.
> 
> That doesn't seem like a viable workaround given that I never had this option enabled in the first place :D

Well, back then it worked, now it doesn't. :)

I've upgraded Ledger Live meanwhile to 2.37.1 (bitcoin app 2.0.1), but the bug is still there. No matter which implementation I choose. This makes basically taproot unusable on Ledger. Well, on a second thought, not only taproot, but everything, because I get this error when I scan for, or want to add new accounts.

**Comment by Blazsar** [comment]
I had the same problem but it worked for me to do the following:

- Reset ledger live (be aware that I said Ledger Live, NOT Ledger device)
- Cleared Ledger Live Cache
- Uninstalled all Apps from my ledger device
- Reinstalled the Bitcoin App on device and initialized it
- Went to accounts on ledger Live and added Bitcoin and it showed my taproot address for adding.

![2022-01-22 - 11-56-13](https://user-images.githubusercontent.com/37079586/150610755-220b69f8-8d4c-4a45-a6d3-0cf544755dc2.png)

**Comment by tyuxar** [comment]
Resetting Ledger Live on macOS also worked for me. What I did:
- removed LL application
- removed all LL related files and directories in my `$HOME`: `find ~ -iname '*ledger*' 2> /dev/null`, go through the list and erase them
- reinstalled LL
- reimported the accounts

I didn't need to turn on the experimental JS implementations for a successful account import.

**Comment by plk** [comment]
I tried all of these resest/reinstalls but still get the same error.

**Comment by memowete** [comment]
I had the same issue. I am not a tech person, however, reading here and elsewhere I figured to clear the Ledger Live Cache first, (Ledger not Connected). Navigated to Settings, Experimental Features, Developer mode - On. Navigated to Manager and connected my Ledger X to verify. Then clicked on Accounts +Add Account. It scanned my devise/ Blockchain and revealed my missing addresses. Can open new accounts now also. Go back to Developer mode and switch off. Hope it helps

**Comment by memowete** [comment]
> I tried all of these resest/reinstalls but still get the same error.

I had the same issue. I am not a tech person, however, reading here and elsewhere I figured to clear the Ledger Live Cache first, (Ledger not Connected). Navigated to Settings, Experimental Features, Developer mode - On. Navigated to Manager and connected my Ledger X to verify. Then clicked on Accounts +Add Account. It scanned my devise/ Blockchain and revealed my missing addresses. Can open new accounts now also. Go back to Developer mode and switch off. Hope it helps

**Comment by premmen2** [comment]
To everyone who got the issue resolved: are you connecting Ledger Live to a full node? For me everything is working until I try to connect Ledger Live with my full node.

**Comment by memowete** [comment]
> To everyone who got the issue resolved: are you connecting Ledger Live to a full node? For me everything is working until I try to connect Ledger Live with my full node.

No, did not connect to full node

**Comment by SpeedForceGN** [comment]
> To everyone who got the issue resolved: are you connecting Ledger Live to a full node? For me everything is working until I try to connect Ledger Live with my full node. 

Are you on the latest software?  This blochchain issue is the second straw for me. First one is that I can not add Algorand to Ledger Live.  Maybe I should look into other choices...

**Comment by SpeedForceGN** [comment]
> To everyone who got the issue resolved: are you connecting Ledger Live to a full node? For me everything is working until I try to connect Ledger Live with my full node. 

Are you on the latest software?  This blochchain issue is the second straw for me. First one is that I can not add Algorand to Ledger Live.  Maybe I should look into other choices...

**Comment by plk** [comment]
> To everyone who got the issue resolved: are you connecting Ledger Live to a full node? For me everything is working until I try to connect Ledger Live with my full node.

Yes, the issue for me is only when I try to connect to my full node.

**Comment by SpeedForceGN** [comment]
So  I  downgrade to version of 2.33.1 ledger live. Once I had this version up and running, I then made sure that developer was turned off (This is the only experimental feature that I activated). I then started the connection to my BITCOIN NODE.  I followed all the steps, downloaded and installed Sat Stack.  I now completed the NODE connection and synced SAT STACK with my accounts.  This took approximately 2 hrs.  After I had things up and running, I left the SAT STACK running and then downloaded and installed 2.38.3.  My connection is still there with my node and Sat Stack is still running.  Then I had to add my accounts back to LEDGER LIVE.  I hope this works for everyone.  

I do need to figure out how to have SAT STACK to autostart on reboot in Windows 11.  Also need to know the differences of the 4 bitcoin accounts.

**Comment by LuckyKey613** [comment]
I had the same error but turned off "Extended account search" in the Experimental Features section in Settings and it worked for me.

**Comment by AnthraxBP** [comment]
I manage to fix this problem it takes me 3 hours to think about it what I do I reset my ledger back to the recovery phase and downgraded from ledger version  2.39.2 to version [v2.38.3]  and synchronize back all my coin. but I still do not test about sending large amounts of funds... that's how I fix the issue of this error 0x6a82 windows 10

**Comment by jagamaster** [comment]
Started to get "Ledger device: UNKNOWN_ERROR (0x6a82)" after upgrading firmware on Ledger Nano S. 
I trying to switch ON "Extended account search" experimental option to import my Bitcoin Native and Legacy account to Ledger live application but get "Ledger device: UNKNOWN_ERROR (0x6a82)" error at account serching step. 
Significant downgrading of Legder Live application doesn't help.

I have another Ledger Nano X with old firmware and it's working fine on the save system.
Using MacOS 10.14.6 Mojave.

**Comment by kraikov** [comment]
Is there a way to get rid of the error with the latest firmware? As a last resort, how to rollback the firmware version? I suppose that's not possible.

**Comment by diwakergupta** [comment]
Any update on this? I'm also running into this issue, both Ledger Live on Desktop and mobile, latest firmware. Simple BTC transfer.

**Comment by mattderron** [comment]
> 

I agree on renaming to specify "connecting to full node", the responses are getting confusing now with references to other seemingly unrelated issues.

Also hoping for a resolution to this soon, ability to connect to a full node on the latest version is important, IMO.

**Comment by goolash761** [comment]
Having the same issues on mine: cannot send transactions, cannot connect to my own node to send transactions, either.  Customer service has been a dead end, telling me to reset, reinstall, etc, but it does not work.

**Comment by banonisking** [comment]
> Im facing the same problem too, I already update my bitcoin apps at ledger manager and update my ledger live desktop version the error came out [(0x6a82)) is there any other way I can get the funds back? can I move it to another wallet with the secret phase phrase? anyone

I also just downloaded the 2.33.1 version and solved the problem. I finally solved it. I tried infinite methods. Finally, I downloaded 2.33.1 and solved it. I hope it will be useful to everyone.

**Comment by banonisking** [comment]
I also just downloaded the 2.33.1 version and solved the problem. I finally solved it. I tried infinite methods. Finally, I downloaded 2.33.1 and solved it. I hope it will be useful to everyone.,
謝謝這裡的大家，我終於可以了

**Comment by banonisking** [comment]
> Having the same issues on mine: cannot send transactions, cannot connect to my own node to send transactions, either.  Customer service has been a dead end, telling me to reset, reinstall, etc, but it does not work.

I also just downloaded the 2.33.1 version and solved the problem. I finally solved it. I tried infinite methods. Finally, I downloaded 2.33.1 and solved it. I hope it will be useful to everyone.

**Comment by vprf06** [comment]
> I had the same problem but it worked for me to do the following:
> 
> * Disable all experimental features first
> * Reset ledger live (be aware that I said Ledger Live, NOT Ledger device)
> * Cleared Ledger Live Cache
> * Uninstalled all Apps from my ledger device
> * Reinstalled the Bitcoin App on device and initialized it
> * Went to accounts on ledger Live and added Bitcoin (toggle the 'Show all address types' switch on) and it showed my taproot address for adding.
> 
> ![2022-01-22 - 11-56-13](https://user-images.githubusercontent.com/37079586/150610755-220b69f8-8d4c-4a45-a6d3-0cf544755dc2.png)

Clearing program cache and turning off  experimental worked for me!

**Comment by jagamaster** [comment]
What exactly Ledger device do you have?
What device firmware version do you have?
What ledger Live version did you installed? OS?

(I am Ledger X with 2.0.2 Secure element firmware, tested with Win10 and MacOS Mojave 10.14.6) with lastest and several versions.

**Comment by ItzSilvx** [comment]
Can we update to the new version once we are done adding the btc account?

**Comment by tomav** [comment]
Hi, is there any update on this? Still have the same issue.
`Ledger device: UNKNOWN_ERROR (0x6a82)`
Ledger Live 2.42.0 on OSX 12.4

**Comment by goolash761** [comment]
I have none.  I ditched their software along with my company and switched to Specter a long time ago.  Specter works great.

Sent from AT&T Yahoo Mail on Android 
 
  On Tue, Jun 7, 2022 at 3:29 PM, Thomas ***@***.***> wrote:   


Hi, is there any update on this? Still have the same issue.
Ledger device: UNKNOWN_ERROR (0x6a82)
Ledger Live 2.42.0 on OSX 12.4
